server/resources.py

```diff
@@ -37,14 +37,20 @@ def wrap(*args, **kwargs):
 class UserRegistration(Resource):
     def post(self):
         data = user_schema.load(request.json)
+        print(request.json)
         
         # Check if email already exists
         if UserModel.find_by_email(data['email']):
+            print('Found this person')
             return {'message': 'User {} already exists'.format(data['email'])}, 409
 
+        print("Didn't find the person")
         new_user = UserModel(
             email = data['email'],
-            credentials = data['credentials']
+            ### I don't think this should be included here
+            ### because we don't instantly authorize with gmail 
+            ### on signup
+            #credentials = data['credentials']
```

Comment:
Need to still pass `credentials` into the model though because it's required in the function right now, but we could make it optional instead ( `credentails=None..`). Database says nullable=True, so seems like that would work?

Reply:
The problem is that the request from the front end doesn't send any credentials, but I'll probably put credentials = None